### PR TITLE
test: write example for ast builder

### PR DESCRIPTION
**Pull request by daengdaengLee**
Hello.

I write example code for AST builder.
The example is same with "hello_world.rs" example, only changed to use AST builder.

Thanks for reading.
Please make comments if I missed something.

### CODE REVIEW

File: pkg/rust/examples/hello_ast_builder.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#[cfg(feature = "sled-storage")]
+mod hello_ast_builder {
+    use gluesql_core::ast_builder::Execute;
+    use {
+        gluesql::{
+            core::ast_builder,
```
Comment (panarch) [inline pkg/rust/examples/hello_ast_builder.rs:6]: Sorry for the late review. Everything else looks nice :)
nit. could you merge this?
we can merge `Execute` into `gluesql::core::ast_builder`
```rust
use {
    gluesql::{
        core::ast_builder::{self, Execute},
    ..
```

**Review by ever0de — APPROVED** [review]

File: pkg/rust/examples/hello_ast_builder.rs (hunk 1 above)
Comment (daengdaengLee) [reply pkg/rust/examples/hello_ast_builder.rs]: Thanks for reply! 

1. I update my fork up to the most recent commit of the origin repository.
2. I merge `Execute` into `gluesql::core::ast_builder`.

Please check when you can. :)

**Review by ever0de — COMMENTED** [review]
Maybe you could add it to the example in the github action CI?

https://github.com/gluesql/gluesql/blob/deb56a429a275520f6c4e7354609f86c247550e1/.github/workflows/rust.yml#L69-L80

**Comment by daengdaengLee** [comment]
> Maybe you could add it to the example in the github action CI?
> 
> https://github.com/gluesql/gluesql/blob/deb56a429a275520f6c4e7354609f86c247550e1/.github/workflows/rust.yml#L69-L80

I add my example to the ci workflow! :)

**Review by panarch — APPROVED** [review]
Looks all nice! Thanks a lot for the contribution 👍 👍

**Comment by daengdaengLee** [comment]
> Looks all nice! Thanks a lot for the contribution 👍 👍

Thanks!!! I'm really excited to make my first contribution to an open-source project. :)

**Pull request merged**